### PR TITLE
Installer text updates

### PR DESCRIPTION
**Pull request by torvista**
Mainly text changes and some minor html things apart from adding this text to give more confidence to users:

Clipboard02

I note that the method of using an array for defining the constants prevents the use of embedding previously-defined constants in the texts. One embedded constant will display, but the addition of a second will display the constant name.
Not an issue, but as I believe the use of arrays is planned generally, something to bear in mind.

### CODE REVIEW

File: zc_install/includes/languages/en_us/main.php [hunk 1]
```diff
@@ -252,19 +253,19 @@
...
 'TEXT_HELP_TITLE_DBCHARSET' => 'Database Character Set',
 'TEXT_HELP_CONTENT_DBCHARSET' => 'Most stores will use utf8mb4 or utf8.<br>If you don\'t have a reason to use something else, use utf8mb4.',
 'TEXT_HELP_TITLE_DBPREFIX' => 'Database Tablename Prefix',
-'TEXT_HELP_CONTENT_DBPREFIX' => 'What is the prefix you would like used for database tables?  Example: <strong>zen_</strong><br><strong class="alert">TIP: Leave empty if no prefix is needed.</strong><br />You can use prefixes to allow more than one store to share the same database.',
+'TEXT_HELP_CONTENT_DBPREFIX' => 'What prefix would you like to use for the Zen Cart&reg; database tables?<br>Historically it was common on shared hosting to have a very limited number of databases offered. Consequently several applications may have shared a single database, using table prefixes to maintain the data separate.<br>Example: <strong>zen_</strong><br><strong class="alert">TIP: Leave empty if no prefix is needed.</strong>',
```
Comment (drbyte) [inline zc_install/includes/languages/en_us/main.php:268]: Is the history lesson still needed?

Is `to maintain the data separate.` good grammar?

File: zc_install/includes/languages/en_us/main.php [hunk 2]
```diff
@@ -18,7 +18,7 @@
 'TEXT_HEADER_MAIN' => 'TIP: The field titles are clickable help links which explain what each field means.',
 'TEXT_INDEX_HEADER_MAIN' => 'TIP: More information may be available by clicking on the <strong>titles</strong> of some errors and warnings.',
 'TEXT_INSTALLER_CHOOSE_LANGUAGE' => 'Installer Language',
-'TEXT_HELP_CONTENT_CHOOSE_LANG' => 'This language selection refers to the installation process only.<br>The Zen Cart&reg; installation files include only the US English language pack.<br>However, Zen Cart&reg; is fully multi-lingual, supporting <em>any</em> language (subject to a language pack having been developed by the community).',
+'TEXT_HELP_CONTENT_CHOOSE_LANG' => 'This language selection refers to the installation process only.<br>The Zen Cart&reg; installation files include only the US English language pack.<br>However, Zen Cart&reg; is fully multi-lingual, supporting <em>any</em> language (subject to a language pack bieing available/having been developed by the community).',
```
Comment (drbyte) [inline zc_install/includes/languages/en_us/main.php:21]: `bieing available/` seems like a misspelling to me ...

Comment (torvista) [reply zc_install/includes/languages/en_us/main.php:21]: Only if you don't have an eight year old with a Nerf Super Speed machine gun in the room.

File: zc_install/includes/languages/en_us/main.php (hunk 1 above)
Comment (torvista) [reply zc_install/includes/languages/en_us/main.php]: I think the history lession is of use. Maybe you don't remember that feeling of total ignorance that a new user has. Prefixes may be ancient history now, but the option is there and anything that reduces the "I don't understand anything" feeling is positive. I think.
I try.

File: zc_install/includes/template/templates/index_default.php [hunk 3]
```diff
@@ -110,7 +110,10 @@
 <?php } ?>
 <?php if (!$hasFatalErrors && !$hasWarnErrors && $hasUpdatedConfigFile) { ?>
     <div class="alert-box success">
-    <?php  echo TEXT_ERROR_SUCCESS_NO_ERRORS; ?>
+    <?php echo TEXT_ERROR_SUCCESS_NO_ERRORS; ?>
+    </div>
+    <div class="alert-box">
+        <?php echo TEXT_UPGRADE_INSTALL_INFO; ?>
```
Comment (drbyte) [inline zc_install/includes/template/templates/index_default.php:116]: Does this properly disappear if no db already exists? 
ie: if no Upgrade option is available, telling them what the non-visible Upgrade button "does" isn't useful, and the text explaining "Clean Install" is probably alarming.

Comment (torvista) [reply zc_install/includes/template/templates/index_default.php:116]: If there are no configures, it all disappears as the error report is based on a configure.
Clipboard01
Maybe better to split the information to a title/popup on each button?

**Pull request merged**